style.css

```diff
@@ -9,16 +9,27 @@
 */
 
 body {
-	margin: 80px;
-	background: #fff;
+   margin: 40px;
 	font-family: 'Poiret One', cursive;
    font-size: 20px;
-   background-color: #F5FFFA;
+   background-color: #F0F8FF;
+}
+
+img {
+     background-image: url("ghettomunda.jpg");
+     background-size: cover;
+     display: inline-block;
+}
+
+div.heading {
```

Comment:
Don't add the element in front of the selector. It's not necessary, just adds bloat to the file and actually might cause confusion later on.